process-engine/consumer_api_contracts
# 📦 Change to eslint and tsc

## Changes

1. Perform builds with pure tsc and remove gulp entirely
2. Switch from TSLint to ESLint
3. Implement the new styleguide we use with eslint
4. Let linter errors cause the Jenkins Builds to fail

## Issues

PR: #71

## How to test the changes

- Running `npm run build` works a great deal faster than before
- Linting is now possible and works as expected
- The Jenkins Builds will also run approx. twice as fast as before
    - Compare job durations [here](https://ci.process-engine.io/blue/organizations/jenkins/process-engine_node-lts%2Fconsumer_api_contracts/activity)
